ruleset/decoders/0520-msexchange-log-decoders.xml

```diff
@@ -1,18 +1,20 @@
 <!--
-  -  MS exchange logs decoders
-  -  Created by Wazuh, Inc.
-  -  Copyright (C) 2015-2021, Wazuh Inc.
-  -  This program is a free software; you can redistribute it and/or modify it under the terms of GPLv2.
+  Copyright (C) 2015-2021, Wazuh Inc.
+-->
+
+<!--
+  Decoders for:
+    Microsoft exchange.
 -->
 
 <decoder name="ECPServerLog">
-    <prematch type="pcre2">,ECP\.Request,</prematch>
+  <prematch type="pcre2">,ECP\.Request,</prematch>
 </decoder>
 
 <decoder name="HTTPProxyLog">
-    <prematch type="pcre2">-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{12},.*,Ecp,</prematch>
+  <prematch type="pcre2">-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{12},.*,Ecp,</prematch>
 </decoder>
 
 <decoder name="OABGenLog">
-    <prematch type="pcre2">,OABGenerator,</prematch>
-</decoder>
\ No newline at end of file
+  <prematch type="pcre2">,OABGenerator,</prematch>
+</decoder>
```

Comment:
Example logs are missing

Reply:
They were removed from the rule files but not incorporated into the decoder file.